Regression/WarpX-tests.ini

```diff
@@ -2992,6 +2992,26 @@ compareParticles = 1
 particleTypes = electrons
 analysisRoutine = Examples/Tests/plasma_lens/analysis.py
 
+[Python_plasma_lens]
+buildDir = .
+inputFile = Examples/Tests/plasma_lens/PICMI_inputs_3d.py
+runtime_params =
+customRunCmd = python3 PICMI_inputs_3d.py
+dim = 3
+addToCompileString = USE_PYTHON_MAIN=TRUE
+cmakeSetupOpts = -DWarpX_DIMS=3 -DWarpX_LIB=ON -DWarpX_APP=OFF
+target = pip_install
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 1
+numthreads = 1
+compileTest = 0
+doVis = 0
+compareParticles = 1
+particleTypes = electrons
+analysisRoutine = Examples/Tests/plasma_lens/analysis.py
```

Comment:
We looked at this (@EZoni @RemiLehe ) and we think we either need to add `outputFile = ...` here or above add:
```
write_dir = '.',
warpx_file_prefix = 'Python_plasma_lens_plt'
```
in the diags of the PICMI file.

Reply:
Thanks! I added the lines to the PICMI file.